# loops

In [1]:
# 1.for loop
# method 1
for i in range(5):
    print(i)

0
1
2
3
4


In [2]:
# method 2
for i in range(1,10,3):
    print(i)

1
4
7


In [3]:
# method1
a='Hello sam'
for i in a:
    print(i,end="")

Hello sam

# 2. while loop

In [5]:
#write a program who print from 1 to 10

i=1   #starting value
n=10   #last value
 while(i<=n):  #condtion
    print(i)
    i=i+1  #increment

1
2
3
4
5
6
7
8
9
10


In [2]:
# write a program to print from 1 to 10 and our 3

a=int(input("enter your number"))
n=int(input("enter your second number"))
y=int(input("enter difference"))
while(a<=n):
    print(a)
    a=a+y

enter your number1
enter your second number10
enter difference2
1
3
5
7
9


In [10]:
#write a program to find sum of square from 1 to n?

i=int(input("enter your number:"))
n=int(input("enter your number:"))
sum=0
while(i<=n):
    sum= sum+(i*i)
    i= i+1
print("Addition =", sum)

enter your number:1
enter your number:5
Addition = 55


In [12]:
# write a program to print only even numbers between 2 to n?
i=int(input("enter your first number:"))
n=int(input("enter last number:"))
while(i<=n):
    print(i)
    i=i+2


enter your first number:2
enter last number:5
2
4


In [14]:
# write a program to find sum of even numbers from 1 to n

i=int(input("enter the number:"))
n=int(input("enter your second number:"))
sum=0
while(i<=n):
    if(i%2==0):
        sum = sum+i
    i=i+1
    
print("sum of even number= ", sum)
    

enter the number:1
enter your second number:10
sum of even number=  30


In [16]:
# write a program to find sum of digits of a given numbers?
 
i=int(input("enter yor number:"))
sum=0
while(i>0):
    sum=sum+(i%10)
    i=i//10
print("sum pf digits=" , sum)

enter yor number:123
sum pf digits= 6


In [2]:
# write a program to find sum of square of digits of a given numbers?

i=int(input("enter number:"))
sum=0
while(i>0):
    sum = sum+(i%10)*(i%10)
    i=i//10
print("Sum of square of each digits =" , sum)

enter number:123
Sum of square of each digits = 14


In [4]:
# write a program to find sum of cube of digits of a given number?
i=int(input("enter number:"))
sum=0
while(i>0):
    sum = sum+(i%10)*(i%10)*(i%10)
    i=i//10
print("Sum of cube of each digits =" , sum)

enter number:123
Sum of cube of each digits = 36


In [2]:
#write a program to print armstrong number?
#Armstrong number ==> if the sum of cube of digits if a values is equal to the value that type of number 
# 153 ==> 1**3+5**3+3**3


i=int(input("enter number:"))
original = i
sum=0
while(i>0):
    sum = sum+(i%10)*(i%10)*(i%10)
    i=i//10
    
if(original==sum):
    print(" number is armstrong number ...." )
else:
    print("number is not armstrong number....")


enter number:153
 number is armstrong number ....


# Task1


write a program to find product of digits of a given number?
 o/p == i=123 1*2*3 =6

In [3]:
# write a program to find reverse of a given number?

i = int(input("enter number:"))
rev=0
while(i>0):
    rev=(rev*10)+(i%10)
    i=i//10
print("reverse =", rev)

enter number:123
reverse = 321
